docs/proof-of-stake/redeem-script.md

```diff
@@ -0,0 +1,39 @@
+# <img class="dcr-icon" src="/img/dcr-icons/QuestionTicket.svg" /> Redeem Script
+
+---
+
+When users sign up with a [Voting Service Provider](/glossary/#voting-service-provider) (VSP), both the user and the VSP jointly create a 1-of-2 multisignature script. This script is known as the redeem script.
+
+The redeem script involves both a public key from the user and a public key from the VSP so that both can vote and revoke the ticket. When the user buys a new ticket, the voting rights are given to a hash of that redeem script.
```

Comment:
Two things here:

- A ticket can only be voted **or** revoked, not both.
- I think these two sentences actually make more sense if the order is reversed.

Suggest changing to

> When a VSP user buys a new ticket, the voting rights are given to a hash of the redeem script. The redeem script involves both a public key from the user and a public key from the VSP, so either are able to vote or revoke the purchased tickets.